com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolume.cs

```diff
@@ -173,22 +173,28 @@ bool IsResponsibleToDrawGizmo()
 
         internal bool ShouldCullCell(Vector3 cellPosition, Vector3 originWS = default(Vector3))
         {
-            var profile = ProbeReferenceVolume.instance.sceneData.GetProfileForScene(gameObject.scene);
-            if (profile == null)
-                return true;
+            var cellSizeInMeters = ProbeReferenceVolume.instance.MaxBrickSize();
```

Comment:
Shouldn't we factor this piece of code inside MaxBrickSize() instead? This way it's always the good value that we get.

Reply:
No, because MaxBrickSize should always be the one loaded. The real time subdiv is coming from profile  because is a special case.